src/collections/resources/tutorials/beginner-guide-to-contributing-to-meshery-and-mesheryctl.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: "Beginner's guide to contributing to Meshery and mesheryctl"
+thumbnail: ../../../assets/images/tutorials/intro-meshery-ctl.png
+category: Service Mesh
+tags: 
+  - Meshery
+  - Tutorial
+author: Navendu Pottekkat
```

Comment:
```suggestion
```
We avoid adding authors to resources. Please make this change in the other files as well.